functions/sendMessageAndRunJanitor.protected.ts

```diff
@@ -51,28 +51,57 @@ export const handler = async (
     const chatChannelJanitor = require(Runtime.getFunctions()['helpers/chatChannelJanitor'].path)
       .chatChannelJanitor as ChatChannelJanitor;
 
-    const channelWebhooks = await context
-      .getTwilioClient()
-      .chat.services(context.CHAT_SERVICE_SID)
-      .channels(channelSid)
-      .webhooks.list();
+    if (conversationSid) {
+      const conversationWebhooks = await context
+        .getTwilioClient()
+        .conversations.services(context.CHAT_SERVICE_SID)
```

Comment:
We just use the default conversations service rather than looking it up via the SID, it has to be the default to be used by flex anyway. Explicitly referencing the API version in the chain helps avoid dependency conflicts in some scenarios Also, explicitly using the `.get` method rather than directly invoking objects as functions makes mocking for unit tests more straightforward.

So this call would be:
```
      const conversationWebhooks = await context
        .getTwilioClient()
        .conversations.v1
        .conversations(conversationSid)
        .webhooks.list();
```

But that's all minor niggles